### DISCUSSION TITLE
Danke!

### DISCUSSION THREAD

**Discussion by SGrosse-Holz** [Rojetto/ReNeo#56] [General]
Habe diese neo Implementierung gerade gefunden, sieht auf den ersten Blick extrem gut aus. Ich habe auch das Gefühl, dass sie einiges schneller ist als der Hotkey-Treiber, den ich bisher verwendet habe (der mir in letzter Zeit auch anderweitig einige Kopfschmerzen gemacht hat). Freue mich riesig über diese Entwicklung!! Made my day:)

**Comment by roteiro** [comment]
Ebenso ein großes Danke! Auf den ersten Blick sind die ganzen nervigen Bugs von Neovars verschwunden. Wie der Vorposter freue ich mich riesig darüber!
